.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+name: Lint
+on:
+  push:
+    branches:
+      - '**'
+      - '!main'
+  pull_request:
+    branches:
+      - '**'
+      - '!main'
+jobs:
+  build:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install
+      - if: ${{ hashFiles('src') != '' }}
+        run: npm run eslint
```

The inclusion of the hash check caused Paper to nuke 5 support threads and support ban 3 noobs.
```suggestion
      - run: npm run eslint
```